### ISSUE TITLE
User and group owner variables are being ignored?

### ISSUE THREAD

**Issue by rfgamaral** [adferrand/docker-letsencrypt-dns#40]
I've tried to set both the environment variable `CERTS_USER_OWNER` and `CERTS_GROUP_OWNER` to `docker:users` but it didn't do anything, the `archive` and `live` folders, along with the cert files inside them were still created with `root:root`.

I've also tried the corresponding `docker` and `users` IDs instead of the strings but it also didn't work...

Am I doing something wrong?

**Comment by adferrand** [comment]
Did you set `CERTS_USER_OWNER=docker:users` or `CERTS_USER_OWNER=docker` and `CERTS_GROUP_OWNER=users` ?

Because in the first case, yes it will fail to set the permissions appropriately. The second case is working from what I see in my tests.

If it is not what you observe while using the second case, could you provide me the container log when it starts ?

**Comment by adferrand** [comment]
Indeed, docker and users are not known from the container. You will need to get their uid and gid from the host, and put uid in CERTS_USER_OWNER, and gid in CERTS_GROUP_OWNER.

Can you do that with the config you posted, and give me back the result and logs?

**Comment by rfgamaral** [comment]
Now it works... I swear I tested the `UID` and `GID` before (because I have many other containers that require that too) and it didn't work, but now it's all good. Sorry about the confusion and thanks for the help.

**Comment by adferrand** [comment]
No problem. I will stay vigilent about this story about files permissions, as we could not find if there were a real flaw in the container or not.

Even if it is only about a misuse from you side, it adds arguments on my side to consider that I really need to refactor the container configuration to make it better readable. 

And your welcome!
